deploy/crds/kubernetes/monitoring.rhobs_alertmanagerconfigs.yaml

```diff
@@ -5,17 +5,15 @@ metadata:
   annotations:
     controller-gen.kubebuilder.io/version: v0.9.2
   creationTimestamp: null
-  name: alertmanagerconfigs.monitoring.coreos.com
+  name: alertmanagerconfigs.monitoring.rhobs
 spec:
-  group: monitoring.coreos.com
+  group: monitoring.rhobs
   names:
     categories:
-    - prometheus-operator
+    - rhobs-prometheus-operator
```

Comment:
Not sure how much the categories are important, but I think we usually call it `obo-prometheus-operator` or?

Reply:
Actually it seems we use/mix both names...

Reply:
This comes from https://github.com/rhobs/obo-prometheus-operator/ and is a result of our rename script. This might be something to track down there, less so in this PR.